library/alloc/src/sync.rs

```diff
@@ -51,8 +51,14 @@ mod tests;
 ///
 /// Going above this limit will abort your program (although not
 /// necessarily) at _exactly_ `MAX_REFCOUNT + 1` references.
+///
+/// This is a global invariant, and also applies when using a compare-exchange loop
+/// (see https://github.com/rust-lang/rust/issues/108706).
```

Comment:
Please do not link an issue in documentation, here, if the issue is not an open and unresolved question. Simply document what is relevant.